vault/raft_test.go

```diff
@@ -35,115 +31,3 @@ func TestFormatDiscoveredAddr(t *testing.T) {
 		}
 	}
 }
-
-// TestRaftDirPath verifies that the Raft data directory path is correctly extracted from the storage configuration.
-// It exercises all execution paths in the RaftDataDirPath to ensure that:
-//   - The path is correctly extracted on a happy path.
-//   - In case of an empty path in config, it returns an empty string and true.
-//   - If there are any issues getting the path, it returns an empty string and false.
-func TestRaftDirPath(t *testing.T) {
```

Comment:
I am curious about why this test is deleted?

Reply:
Yeah, fair question. I deleted it because the new method (of grabbing the directory from the raft config itself) didn't allow for testing the more complicated edge cases. The test would basically end up being: create a core with raft storage and verify that the path is non-empty. Then create a core with non-raft storage, and verify the path is empty. I can add that test, I just wasn't sure how much value it really adds. That being said, it is better than nothing.